2022-02-24 22:25:52,193 INFO    MainThread:26741 [wandb_setup.py:_flush():71] setting env: {}
2022-02-24 22:25:52,193 INFO    MainThread:26741 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-24 22:25:52,194 INFO    MainThread:26741 [wandb_init.py:_log_setup():371] Logging user logs to /home/simon/CDE_UBS/thesis/template/wandb/run-20220224_222552-2hxofnf4/logs/debug.log
2022-02-24 22:25:52,194 INFO    MainThread:26741 [wandb_init.py:_log_setup():372] Logging internal logs to /home/simon/CDE_UBS/thesis/template/wandb/run-20220224_222552-2hxofnf4/logs/debug-internal.log
2022-02-24 22:25:52,194 INFO    MainThread:26741 [wandb_init.py:_jupyter_setup():321] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f21039aa0d0>
2022-02-24 22:25:52,195 INFO    MainThread:26741 [wandb_init.py:init():404] calling init triggers
2022-02-24 22:25:52,195 INFO    MainThread:26741 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-02-24 22:25:52,195 INFO    MainThread:26741 [wandb_init.py:init():460] starting backend
2022-02-24 22:25:52,195 INFO    MainThread:26741 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-24 22:25:52,266 INFO    MainThread:26741 [backend.py:ensure_launched():216] starting backend process...
2022-02-24 22:25:52,342 INFO    MainThread:26741 [backend.py:ensure_launched():222] started backend process with pid: 26785
2022-02-24 22:25:52,346 INFO    MainThread:26741 [wandb_init.py:init():469] backend started and connected
2022-02-24 22:25:52,393 INFO    MainThread:26741 [wandb_run.py:_label_probe_notebook():911] probe notebook
2022-02-24 22:25:52,394 INFO    MainThread:26741 [wandb_run.py:_label_probe_notebook():921] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-02-24 22:25:52,394 INFO    MainThread:26741 [wandb_init.py:init():533] updated telemetry
2022-02-24 22:25:52,504 INFO    MainThread:26741 [wandb_init.py:init():563] communicating current version
2022-02-24 22:25:54,519 INFO    MainThread:26741 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-24 22:25:54,521 INFO    MainThread:26741 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-02-24 22:25:54,738 INFO    MainThread:26741 [wandb_init.py:init():606] starting run threads in backend
2022-02-24 22:25:59,752 INFO    MainThread:26741 [wandb_run.py:_console_start():1810] atexit reg
2022-02-24 22:25:59,753 INFO    MainThread:26741 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.WRAP
2022-02-24 22:25:59,754 INFO    MainThread:26741 [wandb_run.py:_redirect():1721] Wrapping output streams.
2022-02-24 22:25:59,755 INFO    MainThread:26741 [wandb_run.py:_redirect():1745] Redirects installed.
2022-02-24 22:25:59,755 INFO    MainThread:26741 [wandb_init.py:init():633] run started, returning control to user process
2022-02-24 22:26:11,177 INFO    MainThread:26741 [wandb_init.py:_pause_backend():284] pausing backend
2022-02-24 22:26:11,177 INFO    MainThread:26741 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-02-24 22:29:42,426 INFO    MainThread:26741 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 0
2022-02-24 22:29:42,429 INFO    MainThread:26741 [wandb_run.py:_restore():1752] restore
2022-02-24 22:29:42,520 INFO    MainThread:26741 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 761
}

2022-02-24 22:29:42,746 INFO    MainThread:26741 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 761
}

2022-02-24 22:29:42,873 INFO    MainThread:26741 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 11403
}

2022-02-24 22:29:42,978 INFO    MainThread:26741 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 26644
}

2022-02-24 22:29:43,093 INFO    MainThread:26741 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 26644
}

2022-02-24 22:29:43,196 INFO    MainThread:26741 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26644
  total_bytes: 26644
}

2022-02-24 22:29:43,300 INFO    MainThread:26741 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26644
  total_bytes: 26644
}

2022-02-24 22:29:43,402 INFO    MainThread:26741 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26644
  total_bytes: 26644
}

2022-02-24 22:29:43,506 INFO    MainThread:26741 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26644
  total_bytes: 26644
}

2022-02-24 22:29:43,610 INFO    MainThread:26741 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26644
  total_bytes: 26644
}

2022-02-24 22:29:43,715 INFO    MainThread:26741 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26644
  total_bytes: 26644
}

2022-02-24 22:29:44,130 INFO    MainThread:26741 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26644
  total_bytes: 26644
}

2022-02-24 22:29:44,580 INFO    MainThread:26741 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26644
  total_bytes: 26644
}
local_info {
}

